xml/System.Reflection.Metadata/TypeName.xml

```diff
@@ -276,8 +274,7 @@
           <para>This value is computed every time this method gets called, it's not cached.</para>
           <para>There's not really a parallel concept to this in reflection. Think of it</para>
           <para>as the total number of <see cref="T:System.Reflection.Metadata.TypeName" /> instances that would be created if</para>
-          <para>you were to totally deconstruct this instance and visit each intermediate <see cref="T:System.Reflection.Metadata.TypeName" /></para>
-          <para>that occurs as part of deconstruction.</para>
+          <para>you were to totally deconstruct this instance and visit each intermediate <see cref="T:System.Reflection.Metadata.TypeName" /> that occurs as part of deconstruction.</para>
```

Comment:
Here's another example of unncessary `<para>` tags. Rather than pattern matching for hits, I suggest parsing the content of these `<remarks>` and `<summary>` tags, and semantically determining where `<para>` tags would make sense.